src/formio/validators/utils.js

```diff
@@ -0,0 +1,39 @@
+import {parseISO} from 'date-fns';
+
+export const validateBoundaries = (componentType, minBoundary, maxBoundary, value) => {
+  const parsedMinBoundary = minBoundary ? new Date(minBoundary) : null;
+  const parsedMaxBoundary = maxBoundary ? new Date(maxBoundary) : null;
+
+  if (!parsedMinBoundary && !parsedMaxBoundary) {
+    return {isValid: true};
+  }
+
+  const parsedValue = parseISO(value, 'yyyy-MM-dd', new Date());
```

Comment:
shouldn't the format be dependent on the `componentType` too?

Reply:
This actually doesn't seem to be used :thinking: `parseIso` does not have a format argument :thinking: https://date-fns.org/v2.3.0/docs/parseISO#arguments
I will remove these extra args